module/Database/src/Model/SubDecision/Installation.php

```diff
@@ -109,17 +109,35 @@ public function getDischarge(): ?Discharge
         return $this->discharge;
     }
 
-    /**
-     * Get the content.
-     *
-     * Fixes Bor's greatest frustration
-     */
+    protected function getTemplate(): string
+    {
+        return '%MEMBER% wordt geïnstalleerd als %FUNCTION% van %ORGAN_ABBR%.';
+    }
+
+    protected function getAlternativeTemplate(): string
+    {
+        return '%MEMBER% is installed as %FUNCTION% of %ORGAN_ABBR%.';
+    }
+
     public function getContent(): string
     {
-        $member = $this->getMember()->getFullName();
-        $text = $member . ' wordt geïnstalleerd als ' . $this->getFunction();
-        $text .= ' van ' . $this->getFoundation()->getAbbr() . '.';
+        $replacements = [
+            '%MEMBER%' => $this->getMember()->getFullName(),
+            '%FUNCTION%' => $this->getFunction(),
+            '%ORGAN_ABBR%' => $this->getFoundation()->getAbbr(),
+        ];
+
+        return $this->replaceContentPlaceholders($this->getTemplate(), $replacements);
+    }
+
+    public function getAlternativeContent(): string
+    {
+        $replacements = [
+            '%MEMBER%' => $this->getMember()->getFullName(),
+            '%FUNCTION%' => $this->getFunction(), // Has no alternative (like the decision hash).
```

Comment:
For the website we do translate the functions I think?

Reply:
Yes but never in the context of the actual decision. Only when the function is shown separately.

The reason I have not implemented a system for this in GEWISDB is because it requires fundamental changes to how we work with functions. Currently, all board functions are fully customized and only stored in the decisions themselves. Organ functions can be manually added through the settings panel.

Neither of these options allow us to properly specify translations for the functions (unlike the `MeetingTypes` enum). So that would either require us to manually specify them in the translations file like we do for GEWISWEB or we have to start hardcoding them in enums.

I would personally go for the latter solution but at that point I am wondering whether we should just rewrite more of how we deal with the contents of decisions (and also store the non-official translations introduced through this PR). I do not think it is suitable to also do that in this PR.